Assets/Scripts/Enemy/EnemyAI.cs

```diff
@@ -29,13 +29,13 @@ private void Awake()
     {
         _mutant = GetComponent<EnemyMutant>();
         _scale = gameObject.transform.localScale.x;
-        moveTargetPosition = GameObject.Find("Player")?.transform;
         _navMeshAgent = GetComponent<NavMeshAgent>();
         _state = State.Roaming;
     }
 
     private void Start()
     {
+        moveTargetPosition = GameObject.Find("Player(Clone)")?.transform;
```

Comment:
Not the best practice, better to inject it directly or get it from the other object. But, it is fine here, since it is called only once from the Start()